internal/storage/elasticsearch/config/config.go

```diff
@@ -565,6 +638,7 @@ func addLoggerOptions(options []elastic.ClientOptionFunc, logLevel string, logge
 }
 
 // GetHTTPRoundTripper returns configured http.RoundTripper
+
 func GetHTTPRoundTripper(ctx context.Context, c *Configuration, logger *zap.Logger) (http.RoundTripper, error) {
 	if !c.TLS.Insecure {
```

Comment:
why is there early exit here? So if TLS is configured then neither tokens can be passed?